recaf-ui/src/main/java/me/coley/recaf/ui/pane/OutlinePane.java

```diff
@@ -31,10 +31,9 @@
  * @author Matt Coley
  */
 public class OutlinePane extends BorderPane implements ClassRepresentation {
-	private static final SimpleBooleanProperty showTypes = new SimpleBooleanProperty();
-	private static final SimpleBooleanProperty showSynthetics = new SimpleBooleanProperty(true);
-	private final OutlineTree tree = new OutlineTree();
-	private final ClassRepresentation parent;
+	public static final SimpleBooleanProperty showTypes = new SimpleBooleanProperty();
+	public static final SimpleBooleanProperty showSynthetics = new SimpleBooleanProperty(true);
```

Comment:
`public static` makes it feel icky across classes, but I don't really refactoring to do otherwise as worth the effort.